RELEASE/scripts/sl_ascend/sl_tcrs.ash

```diff
@@ -35,39 +35,14 @@ boolean tcrs_consumption()
 {
 	if(!in_tcrs())
 		return false;
-
-	if(sl_beta() && my_adventures() < 5)
+	if(get_property("sl_legacyConsumeStuff").to_boolean())
 	{
-		if(my_inebriety() < 8 && inebriety_left() > 0)
-		{
-			// just drink, like, anything, whatever
-			// find the best and biggest thing we can and drink it
-			sl_autoDrinkOne(false);
-			return true;
-		}
-		if(inebriety_left() > 0 && !get_property("_sl_saving_for_stooper").to_boolean())

```

Comment:
I think this Stooper logic was trash and interacted in hard-to-predict ways with the "it's the end of the day and we're at 15/15 inebriety, let's use our free fights" logic. I didn't want to go to the trouble of fixing it, so I'm ripping it out.